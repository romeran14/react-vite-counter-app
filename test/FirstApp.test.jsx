import { render } from "@testing-library/react"
import { FirstApp } from "../src/FirstApp"

describe('Pruebas firt app', () => { 
    test('should macth snapshot', () => { 
        const title = "hola soy goku"
        
        const { container, getByText} = render( <FirstApp title={title}></FirstApp>)

        expect( container).toMatchSnapshot()
        expect( getByText( title )).toBeTruthy()

        const h1 = container.querySelector('h1')
        console.log(h1.innerHTML)
        expect(h1.innerHTML.trim()).toBe(title)
     })

 })