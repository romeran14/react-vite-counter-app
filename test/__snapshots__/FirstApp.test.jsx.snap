// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas firt app should macth snapshot 1`] = `
<div>
  <h1
    data-testid="test-title"
  >
     
    hola soy goku
     
  </h1>
  <p>
    No hay subtítulo
  </p>
  <p>
    No hay subtítulo
  </p>
  <p>
    Fernando Herrera
  </p>
</div>
`;
